### DISCUSSION TITLE
Feature Request/Upgrade/Modification Ability to Change user notification emails

### DISCUSSION THREAD

**Discussion by NicksMacLife** [sct/overseerr#1891] [Feature Requests]
Is it possible to make the email used to notify a user of a request changeable?

**Comment by TheCatLady** [comment]
No.  If it's a Plex user, they can simply update their email address in Plex, and the new address will get picked up the next time they sign in to Overseerr.

**Reply by ghost** [reply]
The chance of getting all my users to do this is close to zero, unfortunately. It would be great if it could be overridden instead. Probably the last feature missing for me to switch over to overseerr fully since I love everything else about it!
